### DISCUSSION TITLE
Caddy not working - always restart status on docker

### DISCUSSION THREAD

**Discussion by dyesiboy31** [postalserver/postal#1665] [Help with using Postal]
I run this command 

docker run -d \
   --name postal-caddy \
   --restart always \
   --network host \
   -v /opt/postal/config/Caddyfile:/etc/caddy/Caddyfile \
   -v /opt/postal/caddy-data:/data \
   caddy

error on docker logs 
**run: loading initial config: loading new config: http app module: start: tcp: listening on :80: listen tcp :80: bind: address already in use**

![image](https://user-images.githubusercontent.com/65159109/139324516-5ef2aeb7-9e8f-4eb3-87c2-eb86828f1cd0.png)
![image](https://user-images.githubusercontent.com/65159109/139324544-fb09cffc-6faf-4bf8-a093-e63b921dc170.png)

**Comment by willpower232** [comment]
You don't need to run caddy if you're already running nginx or something else outside of docker. You should still be able to proxy to port 5000

**Reply by dyesiboy31** [reply]
Thanks 
![image](https://user-images.githubusercontent.com/65159109/139462282-5177b546-4e8b-4241-aed1-bfb4ed1ca94f.png)

Thanks @willpower232  seems the caddy already up.

**Reply by dyesiboy31** [reply]
![image](https://user-images.githubusercontent.com/65159109/139462765-29f6b39a-0073-4bfd-98f4-e4bbf3f097af.png)
accessing my local via port 5000
http://127.0.0.1:5000/

**Reply by willpower232** [reply]
the caddy image is running but you can see that it isn't using any ports so likely isn't doing anything for you right now.

port 5000 would always work as it is the direct access but you need nginx or caddy to add HTTPS.
